src/Razor/src/Microsoft.AspNetCore.Razor.Language/Legacy/TagHelperParseTreeRewriter.cs

```diff
@@ -162,7 +162,7 @@ public override SyntaxNode VisitMarkupElement(MarkupElementSyntax node)
                 var endTag = (MarkupEndTagSyntax)Visit(node.EndTag);
                 if (endTag != null)
                 {
-                    var tagName = endTag.GetTagName();
+                    var tagName = endTag.GetTagNameWithOptionalBang();
```

Comment:
Are all of these `GetTagNameWithOptionalBang` changes covered by the now unskipped tests?

Reply:
No this was just a rename. `GetTagName` was still including the bang already.